### PR TITLE
Add newlines for lines read from error message in LSP

### PR DESCRIPTION
**Pull request by lask**
This fixes https://github.com/toitlang/toit/issues/252.
image

### CODE REVIEW

File: tools/toitlsp/lsp/compiler/parser.go [hunk 1]
```diff
@@ -119,7 +120,12 @@ func (p *parser) AnalyzeOutput(r io.Reader) (*AnalyzeResult, error) {
 				if line == "*******************" {
 					break
 				}
+				if !first {
+					msg += "\n"
```
Comment (jstroem) [inline tools/toitlsp/lsp/compiler/parser.go:124]: move `first = false` into this branch

**Review by jstroem — APPROVED** [review]
Thanks - good catch

File: tools/toitlsp/lsp/compiler/parser.go [hunk 2]
```diff
@@ -111,6 +111,7 @@ func (p *parser) AnalyzeOutput(r io.Reader) (*AnalyzeResult, error) {
 			}
 
 			var msg string
+			first := true
```
Comment (floitsch) [inline tools/toitlsp/lsp/compiler/parser.go:114]: Maybe switch to `isFirst` but fine with `first` as well.

**Review by floitsch — APPROVED** [review]
LGTM.
thanks.

**Pull request merged**